kfdefs/overlays/osc/osc-cl2/jupyterhub/kfdef.yaml

```diff
@@ -29,17 +29,7 @@ spec:
           name: manifests
           path: radanalyticsio/spark/cluster
       name: radanalyticsio-spark-cluster
```

Comment:
@HumairAK do we want to deploy the spark operator? since this cluster is ocp v4.9

Reply:
this cluster is 4.8.28, but we don't want to deploy it, however I think for this PR we should focus on just restructuring git assets and not live deployment.